README.md

```diff
@@ -52,11 +52,29 @@ df := dataframe.LoadRecords(
 )
 ```
 
+But the new and easier way to generate DataFrames is by 
```

Comment:
This should be worded differently, what is easier for some users it might not be for others. How about:

> Now you can also create DataFrames by loading an slice of arbitrary structs: ...example